src/cloudflare.ts

```diff
@@ -89,6 +89,22 @@ class CloudflareCommHandler extends InngestCommHandler {
 
         case "POST": {
           // Inngest is trying to run a step; confirm signed and run.
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+          const input = await req.json();
+
+          try {
+            this.validateSignature(
+              req.headers.get("x-inngest-signature"),
+              input
+            );
+          } catch (e) {
```

Comment:
For these handlers, do we want to validate the signature in dev too? Ideally, we can trigger functions via the dev server which may not have access to the signing keys.

We could provide an env var like `INNGEST_BYPASS_SIGNATURE=1` that will be picked up here to avoid validating, but then we'd have to always add that in dev which is a pain.

Perhaps we use the `isProd()` check inside `validateSignature()` and only actually validate in production?

Reply:
Completely agree;  this should be prod only and we should warn if not necessary.